hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/datanodes/datanodes.tsx

```diff
@@ -470,7 +517,28 @@ export class Datanodes extends React.Component<Record<string, object>, IDatanode
         </div>
 
         <div className='content-div'>
+          {totalCount > 0 &&
+            <div style={{ marginBottom: 16 }}>
+              <Popconfirm
+                disabled={!hasSelected}
+                placement="topLeft"
+                title={`Are you sure you want Recon to stop tracking these selected ${selectedRowKeys.length} datanodes ?`}
+                icon={
+                  <Icon type='question-circle-o' style={{ color: 'red' }} />
+                }
+                onConfirm={this.popConfirm}
+              >
+                <Tooltip placement="topLeft" title="Remove and stop tracking the DECOMMISSIONED, IN_MAINTENANCE, and DEAD nodes.">
```

Comment:
```suggestion
                <Tooltip placement="topLeft" title="Remove the dead or decommissioned or in maintenance datanodes.">
```

Reply:
@dombizita 
Done changes in latest commit.